custom_components/openmediavault/binary_sensor.py

```diff
@@ -82,54 +82,26 @@ def __init__(self, omv_controller, inst, sensor):
 
         self._state = None
         self._icon = None
-        self._attrs = {ATTR_ATTRIBUTION: ATTRIBUTION}
-
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return f"{self._inst} {self._type[ATTR_LABEL]}"
-
-    @property
-    def device_state_attributes(self):
-        """Return the state attributes."""
-        return self._attrs
-
-    @property
-    def unique_id(self):
-        """Return a unique_id for this entity."""
-        return f"{self._inst.lower()}-{self._sensor.lower()}"
-
-    @property
-    def available(self) -> bool:
-        """Return if controller is available."""
-        return self._ctrl.connected()
-
-    @property
-    def device_info(self):
-        """Return a description for device registry."""
-        info = {
+        self._attr_extra_state_attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
```

Comment:
I'm pretty sure that using attributes like this could make them static

Reply:
aren't they already?

Reply:
they should not be. most of them dont change, but some can change during runtime.